bot/exts/utilities/wtf_python.py

```diff
@@ -0,0 +1,149 @@
+    LOAD = functools.partial(Bot.load_extension)
+    UNLOAD = functools.partial(Bot.unload_extension)
+
+
+class WTFPython(commands.Cog):
+    """Cog that allows getting WTF Python entries from the WTF Python repository."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.get_WTF_python_readme.start()
+
+        self.headers: Dict[str] = dict()
+
+    @tasks.loop(hours=1)
+    async def get_wtf_python_readme(self) -> None:
+        """Gets the content of README.md from the WTF Python Repository."""
+        failed_tries = 0
+
+        for x in range(FETCH_TRIES):
+            async with self.bot.http_session.get(f"{WTF_PYTHON_RAW_URL}README.md") as resp:
+                log.trace("Fetching the latest WTF Python README.md")
+
+                if resp.status == 200:
+                    raw = await resp.text()
+                    self.parse_readme(raw)
+                    log.debug(
+                        "Successfully fetched the latest WTF Python README.md, "
+                        "breaking out of retry loop"
+                    )
+                    break
+
+                else:
+                    failed_tries += 1
+                    log.debug(
+                        "Failed to get latest WTF Python README.md on try "
+                        f"{x}/{FETCH_TRIES}. Status code {resp.status}"
+                    )
+
+        if failed_tries == 3:
+            log.error("Couldn't fetch WTF Python README.md after 3 tries, unloading extension.")
+            action = Action.UNLOAD
+        else:
+            action = Action.LOAD
+
+        verb = action.name.lower()
+        ext = "bot.exts.evergreen.wtf_python"
+
+        try:
+            action.value(self.bot, ext)
+        except (commands.ExtensionAlreadyLoaded, commands.ExtensionNotLoaded):
+            log.debug(f"Extension `{ext}` is already {verb}ed.")
+        else:
+            log.debug(f"Extension {verb}ed: `{ext}`.")
+
+    def parse_readme(self, data: str) -> None:
+        """
+        Parses the README.md into a dict.
+
+        It parses the readme into the `self.headers` dict,
+        where the key is the heading and the value is the
+        link to the heading.
+        """
+        table_of_contents = re.findall(
+            r"\[👀 Examples\]\(#-examples\)\n([\w\W]*)<!-- tocstop -->", data
+        )[0].split("\n")
+        table_of_contents = list(map(str.strip, table_of_contents))
+
+        for header in table_of_contents:
+            match = re.findall(r"\[▶ (.*)\]\((.*)\)", header)
+            if match:
+                self.headers.update(
+                    {
+                        match[0][0]: f"{BASE_URL}{match[0][1]}"
+                    }
+                )
+
+    def fuzzy_match_header(self, query: str) -> Optional[str]:
+        """Returns the fuzzy match of a query if its ratio is above "MINIMUM_CERTAINTY" else returns None."""
+        match, certainty = process.extractOne(query, self.headers.keys())
```

Comment:
This will return a ValueError, as rapidfuzz.process got updated for extractOne where it returns three values, the match itself, the certainty, and the index the string was found.
It doesn't seem like you are using the index so you could just replace it with a wildcard.
This should be updated to:
```suggestion
        match, certainty, _ = process.extractOne(query, self.headers.keys())
```